### DISCUSSION TITLE
Avisoft microphone not recognized by AudioCapture node

### DISCUSSION THREAD

**Discussion by sunmart** [bonsai-rx/bonsai#1379] [General]
Hi there,

I'm having trouble getting the AudioCapture node to recognize my Avisoft microphone. I looked in the device manager and it seems like the Avisoft microphone shows up as a separate device and not under Audio Inputs and Outputs. Is there a way to work around this problem?

Thanks for your help!
Sunaina

**Comment by bruno-f-cruz** [comment]
I don't know if much can be done. I only see two options at this point.
You can ask them to provide generic audio drivers or, if they have one, an SDK that you can try implementing in C#.

**Reply by sunmart** [reply]
Ohh that's unfortunate. I'll see if they can provide other drivers.

Thanks a lot!
